specification/powerplatform/resource-manager/Microsoft.PowerPlatform/preview/2020-10-30-preview/enterprisePolicy.json

```diff
@@ -0,0 +1,514 @@
+            "$ref": "./examples/createOrUpdateEnterprisePolicy.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "enterprisePolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of the EnterprisePolicy."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/EnterprisePolicy"
+            },
+            "description": "Parameters supplied to create or update EnterprisePolicy."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created. The EnterprisePolicy was created successfully.",
+            "schema": {
+              "$ref": "#/definitions/EnterprisePolicy"
+            }
+          },
+          "200": {
+            "description": "OK. The EnterprisePolicy was updated successfully.",
+            "schema": {
+              "$ref": "#/definitions/EnterprisePolicy"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ]
+      },
+      "get": {
+        "tags": [
+          "EnterprisePolicies"
+        ],
+        "operationId": "EnterprisePolicies_Get",
+        "description": "Get information about an EnterprisePolicy",
+        "x-ms-examples": {
+          "Get an EnterprisePolicy": {
+            "$ref": "./examples/getEnterprisePolicy.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "enterprisePolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The EnterprisePolicy name."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
```

Comment:
This reference doesn't exist now and also needs to be updated.

Reply:
thanks! done